### DISCUSSION TITLE
Deployment issues

### DISCUSSION THREAD

**Discussion by Jijiri89** [filamentphp/filament#5987] [General]
Hi please I am deployed my laravel filament app to my hosting plan but the admin dashboard is not working. The frontend works successfully.
What could be the problem?

**Comment by Jijiri89** [comment]
The error I am getting is 404

**Comment by wychoong** [comment]
Check this if you haven’t 

https://filamentphp.com/docs/2.x/admin/installation#deploying-to-production

**Comment by Jijiri89** [comment]
Great, thank you so much for your response. Also,  is it possible to bring
the routes of the filament to web.php where I can route them in the
web.php? At the moment, I wanted to define my own routes for the admin
panel and log in page inside the web.php.

**Comment by wychoong** [comment]
What do you mean define your own route? You want to use url other than `/admin`?

**Comment by Jijiri89** [comment]
OK, what I mean is that, from the filament admin dashboard, if I click on
logout, it should redirect me to my home page. At this point, if I log out
from my admin dashboard, it redirect me to the login page

**Reply by wychoong** [reply]
create your own LogoutResponse and bind it in service provider register

refer this https://github.com/filamentphp/filament/blob/45d8ebce5ff780ba9c270dde890a0b3252d2d537/packages/admin/src/FilamentServiceProvider.php#L92

**Comment by Jijiri89** [comment]
Thanks so much, please I implemented the filamentuser contract but in the
production, I can't login. In my cpanel, I created database and user and
updated my . env file but when I tried logging, it didn't work. They access
denied. Inside my database, the user that I created in the
filament:make-user command is there.

**Reply by wychoong** [reply]
But did you define the `public function canAccessFilament`?

**Comment by Jijiri89** [comment]
Yes, I did.
Public function canAccessFilament() :bool
{
return str_ends_with($this - >email, ***@***.***) :

}
I also imported use Filament \Models \Contracts \FilamentUser;

I implemented as follows
class User extends Authenticatable implements Filamentuser

**Comment by Jijiri89** [comment]
It has worked now. Thanks.
The only problem is the avatar and hasName attributes. Is it not inside the
user.php that I will implement them too? I did and they are not showing in
admin dashboard.

**Comment by ArchieAlvarado20** [comment]
this vid can help. https://www.youtube.com/watch?v=ctPLX04-ui8&list=PL6u82dzQtlfv8fJF3gm42TDHJdtA2NDWT&index=22
watch the first 5mins
